doc/Type/DateTime.pod6

```diff
@@ -252,6 +252,51 @@ Returns an L<Instant|/type/Instant> object based on the invocant.
 
     say DateTime.new('2015-12-24T12:23:00+0200').Instant; # OUTPUT: «Instant:1450952616␤»
 
+=head2 method day-fraction
+
+Defined as:
+
+    method day-fraction(DateTime:D: --> Real:D)
+
+Returns the instant's time as a fraction of a 24-hour day.
```

Comment:
Please capitalize (and maybe link) `Instant`

Reply:
The DateTime object is an instant in the continuum of time. The Instant is a class representing a specific epoch for the instant. (Kind of squishy I know.)